pkg/controller/machine/controller.go

```diff
@@ -203,11 +204,15 @@ func (r *ReconcileMachine) Reconcile(request reconcile.Request) (reconcile.Resul
 				return delayIfRequeueAfterError(err)
 			}
 		}
-
+		metrics.RegisterMachineDelete()
+		deleteStartTime := time.Now()
 		if err := r.actuator.Delete(ctx, cluster, m); err != nil {
 			klog.Errorf("Failed to delete machine %q: %v", name, err)
+			metrics.UpdateDurationFromStart(metrics.Delete, deleteStartTime)
+			metrics.RegisterFailedMachineDelete(err.Error())
```

Comment:
This doesn't seem like a useful metric to capture here.  We'll just be spamming in case of async deletes.

Reply:
should we add a timer with the machine and create metric if machine still there after long enough?